raiden/ui/cli.py

```diff
@@ -259,7 +259,7 @@ def options(func):
                     'manually, i. e. "--pathfinding-service-address" set to a value other than '
                     '"auto", this argument must be set to a valid EIP55 address.'
                 ),
-                type=str,
+                type=ADDRESS_TYPE,
```

Comment:
This converts the argument to `Address`, see `AddressType`, so the argument in `run_app` needs change.